_includes/portfolio.html

```diff
@@ -64,7 +64,7 @@
 					<h3>{{ item.heading | markdownify | remove: "<p>" | remove: "</p>" }}</h3>
 					{% if item.meta %}<span class="meta">{{ item.meta }}</span>{% endif %}
 				</div>
-				<img src="{% if item.thumbnail %}{{ item.thumbnail }}{% endif %}" alt="" />
+				<img src="{% if item.categories contains "plugin-thumbnail" %}/plugins/images/thumbnails/plugin.png{% else %}{{ item.thumbnail }}{% endif %}" alt="" />
```

Comment:
Hmm.  Yeah, that would work pretty good.  I thought of something similar, but it sounded a lot more cumbersome in my head.  :sweat:

I was too focused on the idea of tying the thumbnail to the *plugin* category specifically.

Actually, it would probably work a lot *better* if I changed the "plugin" style to simply be a "Default Thumbnail" style. :unamused:

Maybe I can make a stand-in graphic for Themes as well (and of course make a better version for Plugins).  And the idea of a blank fallback image would also help prevent breaking if there's no thumbnail.

Alright, taking it in a new direction then. :smile:

That's just what I needed. :+1: